### PR TITLE
[Testing] LA.UM.6.4.r1-06500-8x98.0

### PR DESCRIPTION
**Pull request by humberos**
Up to LA.UM.6.4.r1-06500-8x98.0

Needs testing!

I am running it on suzu from now.

### CODE REVIEW

File: arch/arm/configs/sdm660-perf_defconfig [hunk 1]
```diff
@@ -17,8 +17,6 @@ CONFIG_CGROUP_FREEZER=y
 CONFIG_CPUSETS=y
 CONFIG_CGROUP_CPUACCT=y
 CONFIG_CGROUP_SCHEDTUNE=y
-CONFIG_MEMCG=y
```
Comment (oshmoun) [inline arch/arm/configs/sdm660-perf_defconfig:20]: should these be also applied to our defconfigs or are these specific to sdm660? according to their Kconfig entries they seem helpful actually...
pinging @kholk (sorry!)

**Comment by kholk** [comment]
Sorry for what? :D
No just leave our configurations as they are, for now. We will revisit them on the next kernel or the next Android upgrade.

**Comment by oshmoun** [comment]
just being a polite pinger :sweat_smile: 
cool, so no need for a change to defconfig.
Tested this PR as is, and works fine on kugo (as expected since it worked on suzu).
Will test keyaki when there's time.

**Comment by kholk** [comment]
Hahaha! Yeah, I know :D
Btw, the PR has to be tested particularly on Yoshino, since there are changes in IPA code. I may be able to build tomorrow for Maple... After finishing a piece of code started yesterday :p

**Comment by jerpelea** [comment]
it looks fine on lilac but testing is needed

I think that we should implement a 7 day rule
we should allow 7 days of testing for any code that does not fix a critical issue
what do you think @kholk @oshmoun @humberos @alviteri @BuzzBumbleBee  @EnJens

**Comment by oshmoun** [comment]
sure, sounds fine
1 day down, 6 to go :)

**Comment by jerpelea** [comment]
not really 3 down 4 to go

**Comment by tomgus1** [comment]
Been running on Suzu past 5 days, no issues thus far.

**Pull request merged**